pkg/kubectl/cmd/portforward.go

```diff
@@ -192,6 +206,16 @@ func (o PortForwardOptions) RunPortForward() error {
 		}
 	}()
 
+	if o.Timeout > 0 {
+		go func() {
+			select {
+			case <-time.After(o.Timeout):
+				fmt.Fprintf(o.errOut, "info: timed out request after %v\n", o.Timeout)
+				o.StopChannel <- struct{}{}
```

Comment:
this looks like its making a time bomb for the command.  Why are you doing this again?

Reply:
Per [this bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1543797), there is no way to timeout `kubectl port-forward`, as the `--request-timeout` flag was not being honored